shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -168,37 +197,36 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
 }
 
 - (void)applyBlurBackdropFilters:(NSMutableArray<PlatformViewFilter*>*)filters {
```

Comment:
Use `NSArray` instead since you're not mutating it?

Reply:
done